api_contracts/contracts/search/AdvancedSearch.md

```diff
@@ -30,7 +30,6 @@ This endpoint returns a list of the top results of specific categories sorted by
             + `county`
             + `district`
             + `federal_accounts`
-            + `product_service`
             + `countries`
             <!-- + `states_and_territories` -->
```

Comment:
We should work with backend to determine variable names for the new categories and then add them here.